doc/rst/technotes/gpu.rst

```diff
@@ -178,6 +180,8 @@ code for and interacts with GPUs. These variables include:
   ``hipcc``.  This variable is unused if not targeting AMD GPUs. For more
   information, see the `Vendor Portability`_ section.
 
+* ``CHPL_LOCALE_MODEL`` --- must be set to ``gpu`` to enable GPU support.
+
```

Comment:
Could you also take out the first sentence in the paragraph at the beginning of this section and rephrase the rest appropriately. If you would like, we could also put this bullet at the top of the list since it is the first step in making the rest of it work.

Additionally, it would be helpful if we mention here that:
1. Changing this variable requires rebuilding Chapel for it to take effect.
2. Add a sentence to link to the page with more information about `CHPL_LOCALE_MODEL`. Linking can be done as follows: 
```
:ref:`readme-chplenv.CHPL_LOCALE_MODEL`
```